lib/seed_helper.rb

```diff
@@ -153,6 +153,15 @@ def report_templates
       Dir["#{Rails.root}/app/views/unattended/report_templates/*.erb"]
     end
 
+    def plugin_templates
+      dirs = Foreman::Plugin.all.map(&:template_dirs)
+      files = dirs.flatten.compact.map do |dir|
+        Dir.glob(dir + '/**/*.erb')
+      end
+
+      files.flatten
```

Comment:
Since Ruby 2.7 you can use `filter_map` instead of `compact.map`. Also, instead of `flatten.map` you can use `flat_map` which is also faster. I wouldn't expect `nil` and I'd optimize it like this:
```suggestion
      globs = Foreman::Plugin.all.map(&:template_dirs).flat_map { |dir| File.join(dir, '**', '*.erb') }
      Dir[*globs]
```